### PR TITLE
Docs: issue-3501: reorganize updated IoT docs

### PR DESCRIPTION
**Pull request by jenmalloy**
### Type of change
Documentation

### Description
Follow on to PR #3447: reorganize new IoT docs

### Checklist
- [ ] Update/write design documentation in `./documentation/design`
- [ ] Write tests and make sure they pass
- [X] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by jenmalloy** [comment]
Local docs build: http://file.rdu.redhat.com/~jmalloy/openshift/#iot-getting-started-messaging-iot-iot

File: documentation/assemblies/assembly-iot-service-admin-guide.adoc [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
-= IoT for service administrators 
+= IoT for service administrators
+
+Service administrators typically install and configure the Internet of Things (IoT) services for {ProductName}.
+
+include::../modules/proc-iot-installing-services.adoc[leveloffset=+1]
+
+include::assembly-iot-configuring-sigfox.adoc[leveloffset=+1]
+
+include::assembly-iot-creating-device.adoc[leveloffset=+1]
```
Comment (dejanb) [inline documentation/assemblies/assembly-iot-service-admin-guide.adoc]: This should go in the tenant guides. Tenants use device registry to manage (create, update, delete) devices

File: documentation/assemblies/assembly-iot-tenant-guide.adoc [hunk 2]
```diff
@@ -7,49 +7,23 @@
 [id='iot-tenant-{context}]
 = IoT for messaging tenants
 
+Messaging tenants are typically responsible for sending telemetry and events using IoT supported protocols.
```
Comment (dejanb) [inline documentation/assemblies/assembly-iot-tenant-guide.adoc]: I would suggest to use something more broad here (as we'll add more material in the future).

"Messaging tenants are typically responsible for managing devices and writing device and back-end applications of IoT solutions"

File: documentation/assemblies/assembly-iot-getting-started.adoc [hunk 3]
```diff
@@ -9,24 +9,19 @@
 
 The following information describes how to set up and manage {ProductName} IoT features.
 
+include::../modules/con-iot-concepts.adoc[leveloffset=+1]
```
[the remark this answers is not in the document]
Comment (jenmalloy) [reply documentation/assemblies/assembly-iot-getting-started.adoc]: @dejanb, I pulled that section out of the assembly-iot-guide.adoc file, so now it's just in this getting started section.

**Comment by jenmalloy** [comment]
@dejanb, I pushed edits to address your comments. Can you review again/approve? Thanks!

File: documentation/assemblies/assembly-iot-tenant-guide.adoc [hunk 4]
```diff
@@ -7,49 +7,27 @@
...
+Messaging tenants are typically responsible for managing devices and writing device and back-end applications for IoT solutions.
 
-== Customer applications
+include::assembly-iot-configuring-sigfox.adoc[leveloffset=+1]
 
-include::../modules/proc-iot-configure-hono-cli.adoc[leveloffset=+1]
+include::assembly-iot-creating-device.adoc[leveloffset=+1]
 
-:api: telemetry
+== Consumer applications
```
Comment (dejanb) [inline documentation/assemblies/assembly-iot-tenant-guide.adoc]: I think we don't need this headline anymore, as it's empty.

**Pull request merged**